10.0/apache/docker-entrypoint.sh

```diff
@@ -1,9 +1,45 @@
 #!/bin/bash
 set -e
 
-if [ ! -e '/var/www/html/version.php' ]; then
-    tar cf - --one-file-system -C /usr/src/nextcloud . | tar xf -
+# version_greater A B returns whether A > B
+function version_greater() {
+	[[ "$(printf '%s\n' "$@" | sort -V | head -n 1)" != "$1" ]];
```

Comment:
Yeah, that's because `sort -V` is a GNU coreutils thing. I really am not a fan of that script you linked, GNU's version checking is much more sophisticated. Are we basing our images on `alpine`? I feel like we shouldn't because of the other issues that Alpine container images have...